RELEASE.md

```diff
@@ -1,4 +1,4 @@
-liboqs-java version 0.1.0
+liboqs-java version 0.1.1
```

Comment:
Thoughts on making this 0.2.0 instead?

Reply:
I think since it's a notable API change it's worth bumping to 0.2.0.

Reply:
Okay, I agree.. I will move it to 0.2.0